docs/changelog.md

```diff
@@ -1,5 +1,7 @@
 0.15.0.0
 
+Upgrade instructions: When upgrading to 0.15.0.0 or above, you must re-install all packages to take advantage of the new persistent pipx metadata files introduced in this release. These metadata files store pip specification values, injected packages, any custom pip arguments, and more in each main package's venv. You can do this by running `pipx reinstall-all` or `pipx uninstall-all`, then reinstalling manually.
```

Comment:
What about adding "from a pre-0.15.0.0 version":
> When upgrading to 0.15.0.0 or above from a pre-0.15.0.0 version,

Otherwise maybe it sounds like you have to do this every time you upgrade for future versions as well.